.github/workflows/breaking-change-pr-release.yaml

```diff
@@ -4,10 +4,45 @@ on:
   pull_request:
     branches:
       - "master"
+    types:
+      - opened
+      - edited
+      - synchronize
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number }}
+  cancel-in-progress: true
```

Comment:
Please, add a description to the PR.

Reply:
Sorry I've marked it as Ready for Review only to validate the desired workflow is being triggered, as we want to only happen when the PR is ready for review

Reply:
I am still running some validations